sdk/appcenter-crashes/src/main/java/com/microsoft/appcenter/crashes/Crashes.java

```diff
@@ -693,6 +762,8 @@ private void processPendingErrors() {
                 }
             }
         }
+        mHasReceivedMemoryWarningInLastSession = isMemoryRunningLevelWasReceived(SharedPreferencesManager.getInt(PREF_KEY_MEMORY_RUNNING_LEVEL, -1));
```

Comment:
if we start being disabled we should not do that and simply remove the state

Reply:
It's called only if enabled on start. see https://github.com/microsoft/appcenter-sdk-android/pull/1216/files#diff-f948cee626f7c221be3d4eee50bedcb8R451

+ Removing is already done at this point - we remove it on disable